libraries/tlv-account-resolution/src/state.rs

```diff
@@ -39,15 +46,6 @@ fn de_escalate_account_meta(account_meta: &mut AccountMeta, account_metas: &[Acc
     }
 }
 
-/// Helper to convert an `AccountInfo` to an `AccountMeta`
-fn account_meta_from_info(account_info: &AccountInfo) -> AccountMeta {
-    AccountMeta {
-        pubkey: *account_info.key,
-        is_signer: account_info.is_signer,
-        is_writable: account_info.is_writable,
-    }
-}
```

Comment:
This helper was moved into tests, but we can just keep it, referring to https://github.com/solana-labs/solana-program-library/pull/5182/files#r1332852715